src/amber/support/predicates.cr

```diff
@@ -0,0 +1,174 @@
+module Amber
+  module Support
+    module Predicates
+      module String
+        macro included
```

Comment:
Actually no, the intention of the macro included is to extend to monkey path the string class with new predicte methods